frontend/src/pages/tracker-report.module.scss

```diff
@@ -0,0 +1,284 @@
+@import "../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.wrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+@keyframes fadein {
+  from {
+    opacity: 0;
+  }
+  50% {
+    // Let the loading screen stay hidden for 200ms;
+    // if it gets removed in that time (because the report has initialised),
+    // showing a "loading" message is just distracting:
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.loading {
+  padding: $spacing-xl;
+  height: 100%;
+  background-color: $color-grey-05;
+  animation-name: fadein;
+  animation-duration: 400ms;
+}
+
+.load-error {
+  padding: $spacing-xl;
+  height: 100%;
+  background-color: $color-grey-05;
+}
+
+.report-wrapper {
+  background-color: $color-grey-05;
+  flex: 1 0 auto;
+  display: flex;
+  justify-content: center;
+
+  .report {
+    width: 100%;
+    max-width: $content-xl;
+    padding: $spacing-xl;
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-md;
+  }
+
+  .logo {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: $spacing-xs;
+
+    .logomark {
+      max-height: 35px;
+    }
+    .logotype {
+      max-height: 20px;
+    }
+  }
+
+  h1 {
+    @include text-title-2xs;
+    font-family: $font-stack-firefox;
+    padding: $spacing-md 0;
+    border-bottom: 1px solid $color-grey-20;
+  }
+
+  .meta {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-md;
+    padding: $spacing-lg;
+
+    @media screen and #{$mq-md} {
+      flex-direction: row;
+      gap: $spacing-2xl;
+      padding: $spacing-lg $spacing-2xl;
+    }
+
+    dt {
+      color: $color-grey-40;
+    }
+
+    dd {
+      font-weight: 600;
+      color: $color-grey-50;
+    }
+
+    .from {
+      flex: 1 0 auto;
+
+      dd {
+        text-overflow: ellipsis;
+        overflow: hidden;
+        max-width: $content-sm;
+      }
+    }
+  }
+
+  .trackers {
+    background-color: $color-white;
+    border-radius: $border-radius-md;
+    display: flex;
+    flex-direction: column;
+    box-shadow: $box-shadow-sm;
+    // We typically avoid margins because they affect the positioning of other elements,
+    // but since box-shadow is visible below the element, the bottom margin lets
+    // us pretend like it's part of the element:
+    margin-bottom: $spacing-md;
+    padding: $spacing-lg;
+    gap: $spacing-md;
+
+    @media screen and #{$mq-md} {
+      padding: $spacing-lg $spacing-2xl;
+      gap: $spacing-xl;
+    }
+
+    h2 {
+      @include text-title-xs;
+      font-family: $font-stack-firefox;
+      font-weight: 500;
+    }
+
+    table {
+      // The table heading only exists for screen readers:
+      thead {
+        position: absolute;
+        left: -1000000px;
+        width: 1px;
+        height: 1px;
+        overflow: hidden;
+      }
```

Comment:
Yeah, it feels dirty, right? I got this from here: https://webaim.org/techniques/css/invisiblecontent/#offscreen

I just took a look at how Tailwind does it, which is as follows:

```css
.sr-only {
	position: absolute;
	width: 1px;
	height: 1px;
	padding: 0;
	margin: -1px;
	overflow: hidden;
	clip: rect(0, 0, 0, 0);
	white-space: nowrap;
	border-width: 0;
}
```

Using `clip` feels a little bit better, as it doesn't include the arbitrarily high number, so I've switched to that (and `clip-path`, since MDN says `clip` is deprecated), though of course it's still somewhat of a workaround. Also added a link to the above article, in case it gets updated with a more reliable technique in the future: 1d2a039a

Happy to take other suggestions of course!